apps/sms/test/unit/thread_list_ui_test.js

```diff
@@ -1857,4 +1857,167 @@ suite('thread_list_ui', function() {
       assert.ok(thread1.querySelector('input').checked);
     });
   });
-});
+
+  suite('contextmenu handling > Long press on the thread', function() {
+    var draft,
+        threads = [{
+          id: 1,
+          date: new Date(2013, 1, 2),
+          unread: true
+        }, {
+          id: 2,
+          date: new Date(2013, 1, 0),
+          unread: false
+        }];
+
+    setup(function(done) {
+      MockOptionMenu.mSetup();
```

Comment:
you should not need this, it should be executed by MocksHelper (unless this file is not setup like others ?)